ouroboros-network/src/Ouroboros/Network/PeerSelection/Churn.hs

```diff
@@ -90,7 +92,9 @@ peerChurnGovernor tracer deadlineChurnInterval bulkChurnInterval
                    ChurnModeNormal ->
                        decrease $ targetNumberOfActivePeers base
                    ChurnModeBulkSync ->
-                       min 1 (targetNumberOfActivePeers base - 1)
+                       min 1 (targetNumberOfActivePeers base - 1),
+          targetNumberOfActiveBigLedgerPeers =
+              decrease $ targetNumberOfActiveBigLedgerPeers base
```

Comment:
TODO: to be discussed too

Reply:
@karknu I changed it so that we have just one big ledger peer in the bulk sync mode.

Reply:
#4609

Reply:
@coot I think that we should split up the active churning into two parts, one for bigLedger peers and one for normal peers.

Currently we only wait for 1s between decreasing the target and increasing. We have seen cases during high load where this didn't trigger a churn because the governor didn't notice the changed target before it was restored. If we first churn bigLedger peers and then churn normal peers we can afford to wait longer, possibly even wait for confirmation that the governor has taken action, before we restore the targets.

Reply:
Done.